.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+    workflow_dispatch:
+
+jobs:
+    validation:
+        name: "Validation"
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php: '7.4'
+
+        env:
+            APPLICATION_ENV: devtest
+            APPLICATION_STORE: DE
+            PROJECT: crefo-pay
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Configure sysctl limits
+              run: |
+                  sudo swapoff -a
+                  sudo sysctl -w vm.swappiness=1
+                  sudo sysctl -w fs.file-max=262144
+                  sudo sysctl -w vm.max_map_count=262144
+            - name: Composer get cache directory
+              id: composer-cache
+              run: |
+                  echo "::set-output name=dir::$(composer config cache-files-dir)"
+            - name: Composer cache
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+            - name: Composer validate
+              run: composer validate
+
+            - name: Composer install
+              run: |
+                  composer --version
+                  composer install
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+                  extensions: mbstring, intl
+                  tools: composer:v2
+
+            - name: Run CodeStyle checks
+              run: composer cs-check
```

Comment:
cs usually comes last (after stan)

Reply:
Also, where are the other jobs? for actual test runs?
I only see validation for stan and cs, no tests.